MSCL/source/mscl/MicroStrain/MIP/MipTypes.h

```diff
@@ -1677,6 +1679,25 @@ namespace mscl
 
     typedef std::vector<DeviceCommPort> CommPortInfo;
 
+    //API Struct: EventActionInfo
+    //  Information about an event action
+    struct EventActionInfo
+    {
+        //API Variable: type
+        //  Configured action type
+        uint8 type;
```

Comment:
this type will need to be an EventActionConfiguration::Type. Class will need to be moved to ExposedInertialTypes and this is blocked until svn/event_action_configuration is merged